### PR TITLE
TAJO-1653: Add 'INSERT INTO VALUES' statement for OLTP-like storages.

### PR DESCRIPTION
**Pull request by hyunsik**

### CODE REVIEW

**Comment by jihoonson** [comment]
Hi @hyunsik, this patch emits the following error.
```
pgsql_db> insert into test values (1::int8, 'name1');
ERROR: org.apache.tajo.exception.NotImplementedException: not implemented feature: org.apache.tajo.storage.jdbc.JdbcTablespace::verifySchemaToWrite
```

Here is the log.
```
2015-10-23 13:19:03,298 WARN org.apache.tajo.plan.verifier.VerificationState: org.apache.tajo.master.GlobalEngine:verifyInsertTableSchema(320) causes: org.apache.tajo.exception.NotImplementedException: not implemented feature: org.apache.tajo.storage.jdbc.JdbcTablespace::verifySchemaToWrite
2015-10-23 13:19:03,298 ERROR org.apache.tajo.master.GlobalEngine:
Stack Trace:
org.apache.tajo.exception.TajoRuntimeException: org.apache.tajo.exception.NotImplementedException: not implemented feature: org.apache.tajo.storage.jdbc.JdbcTablespace::verifySchemaToWrite
        at org.apache.tajo.storage.jdbc.JdbcTablespace.verifySchemaToWrite(JdbcTablespace.java:160)
        at org.apache.tajo.master.GlobalEngine.verifyInsertTableSchema(GlobalEngine.java:315)
        at org.apache.tajo.master.GlobalEngine.createLogicalPlan(GlobalEngine.java:287)
        at org.apache.tajo.master.GlobalEngine.executeQuery(GlobalEngine.java:190)
        at org.apache.tajo.master.TajoMasterClientService$TajoMasterClientProtocolServiceHandler.submitQuery(TajoMasterClientService.java:303)
        at org.apache.tajo.ipc.TajoMasterClientProtocol$TajoMasterClientProtocolService$2.callBlockingMethod(TajoMasterClientProtocol.java:675)
        at org.apache.tajo.rpc.BlockingRpcServer$ServerHandler.channelRead0(BlockingRpcServer.java:100)
        at org.apache.tajo.rpc.BlockingRpcServer$ServerHandler.channelRead0(BlockingRpcServer.java:61)
        at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:105)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
        at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:244)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
        at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:86)
        at org.apache.tajo.rpc.MonitorServerHandler.channelRead(MonitorServerHandler.java:70)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
        at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:846)
        at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
        at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:511)
        at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:468)
        at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:382)
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:354)
        at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:111)
        at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.tajo.exception.NotImplementedException: not implemented feature: org.apache.tajo.storage.jdbc.JdbcTablespace::verifySchemaToWrite
        ... 29 more
```

**Comment by hyunsik** [comment]
This issue is for Tajo QueryExecutor layer. So, it works with Hbase that already supports INSERT. In contrast, JDBC storage does not support INSERT.

File: tajo-sql-parser/src/main/antlr4/org/apache/tajo/parser/sql/SQLParser.g4 [hunk 1]
```diff
@@ -951,7 +951,7 @@ parenthesized_boolean_value_expression
 
 /*
 ===============================================================================
-  7.2 <row value expression>
+  7.2 <row value expression> (p293)
```
Comment (jihoonson) [inline tajo-sql-parser/src/main/antlr4/org/apache/tajo/parser/sql/SQLParser.g4]: Would you explain what ```p293``` means?

**Comment by jihoonson** [comment]
Oh, ok. I misunderstood due to the title. 
Here is my +1.

**Comment by hyunsik** [comment]
Thank you! I just committed.

**Pull request closed without merging**